source/tests/system/nirfmxbt_driver_api_tests.cpp

```diff
@@ -211,6 +258,48 @@ TEST_F(NiRFmxBTDriverApiTests, TxpBasicFromExample_DataLooksReasonable)
   EXPECT_GT(fetched_powers_response.peak_to_average_power_ratio_maximum(), 0.0);
 }
 
+// Note: there aren't any complex attributes in attributes.py, but this at least exercises the code.
+TEST_F(NiRFmxBTDriverApiTests, SetAttributeComplex_ExpectedError)
```

Comment:
RFmxBT has several APIs that actually use complex numbers: `AnalyzeIQ1Waveform`, `ModAccFetchConstellationTrace`. We should write an example-based test that uses at least one of those. @DavidCurtiss may be able to help generate those. And, if they use TDMS files, @reckenro was looking at pulling out some of that code/data for a different test.

Reply:
It's actually @danestull who is working with TDMS -> JSON files along with a test that uses complex numbers. He posted a PR for the tests using the JSON waveform files he converted from TDMS here: #513

Reply:
Good call. I've created a new work item for this: [AB#1835516](https://ni.visualstudio.com/DevCentral/_workitems/edit/1835516).